src/component/page/landing-page/LandingPage.tsx

```diff
@@ -27,7 +37,15 @@ export const LoginPage = (): React.ReactElement => {
 				som har en avtale om bruk som har tilgang til tjenesten.
 			</AlertStripeAdvarsel>
 
-			<a className={cls('knapp', 'knapp--hoved', globalStyles.lenkeKnapp)} href={loginUrl}>Logg inn</a>
+			<Show if={props.view === LandingPageView.IKKE_TILGANG}>
+				<AlertStripeFeil className={cls(styles.alertstripe)}>
+					Du har ikke tilgang til tjenesten.
+				</AlertStripeFeil>
+			</Show>
+
+			<Show if={props.view === LandingPageView.LOGIN}>
+				<a className={cls('knapp', 'knapp--hoved', globalStyles.lenkeKnapp)} href={loginUrl}>Logg inn</a>
```

Comment:
Tenker i utgangspunktet at denne knappen godt kan være der uansett, fordi det ikke skader og for å ha mindre kompleksitet

Reply:
Hadde en prat med Linn om det. Grunnen til at den fjernes er for at man ikke skal havne i en loop hvis man logger inn og ikke har tilgang til løsningen